src/azure-cli/azure/cli/command_modules/acr/connected_registry.py

```diff
@@ -394,22 +394,21 @@ def _get_install_info(cmd,
                       registry_name,
                       regenerate_credentials,
                       resource_group_name=None):
-    registry, resource_group_name = validate_managed_registry(
+    _, resource_group_name = validate_managed_registry(
         cmd, registry_name, resource_group_name)
     connected_registry = acr_connected_registry_show(
         cmd, client, connected_registry_name, registry_name, resource_group_name)
     parent_gateway_endpoint = connected_registry.parent.sync_properties.gateway_endpoint
+    if parent_gateway_endpoint is None or parent_gateway_endpoint == '':
+        parent_gateway_endpoint = "<parent gateway endpoint>"
     parent_id = connected_registry.parent.id
-    sync_token_name = connected_registry.parent.sync_properties.token_id.split('/tokens/')[1]
     if parent_id:
-        parent = parent_id.split('/connectedRegistries/')[1]
-        parent = acr_connected_registry_show(
-            cmd, client, parent, registry_name, resource_group_name)
-        parent_registry_endpoint = parent.login_server.host
+        parent_endpoint_protocol = "https"
     else:
-        parent_registry_endpoint = registry.login_server
+        parent_endpoint_protocol = "<http or https>"
```

Comment:
Is parent_id is not null, is parent is cloud registry or connected registry?

Reply:
If parent_id is not null, parent is a connected registry.

Reply:
Fixed.